grow/ui/sass/tool_ga_inspect.sass

```diff
@@ -0,0 +1,19 @@
+.grow__tool__ga
+  &--active
+    [data-g-event]
+      background: repeating-linear-gradient(-45deg, #FBE9E7, #FFCCBC 5px, white 5px, white 10px)
+
+      &:hover::after
+        background: #E64A19
+        border-radius: .25em
+        bottom: 50px
+        color: #ffffff
+        content: '[Event] ' attr(data-g-event) ' [Action] ' attr(data-g-action) ' [Label] ' attr(data-g-label)
```

Comment:
The analytics tracking or experimentation is one I can think of off the top of my head.

Right now unless a website uses `autotrack.js` (which is from Google, not even related to Grow or open source) the `data-g-event` stuff won't actually do anything. So, my thinking is we add support for some equivalent functionality in airkit, that does something similar to `autotrack.js` and then we add support for it in Grow's UI.